microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/v10/OpenApi10OnJaxRsAnnotationsTest.java

```diff
@@ -0,0 +1,221 @@
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientRelatedException;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.yaml.snakeyaml.Yaml;
+
+import io.restassured.RestAssured;
+
+/**
+ * Test cases for MP OpenAPI v. 1.0 annotations on JAX-RS components (migrated from RHOAR QE TS)
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class OpenApi10OnJaxRsAnnotationsTest {
+
+    private final static String DEPLOYMENT_NAME = OpenApi10OnJaxRsAnnotationsTest.class.getSimpleName();
+
+    static String openApiUrl, resourceUrl;
+    static OnlineManagementClient onlineManagementClient;
+
+    @BeforeClass
+    public static void setup() throws ManagementClientRelatedException, ConfigurationException {
+        openApiUrl = OpenApiDeploymentUrlProvider.composeDefaultOpenApiUrl();
+        resourceUrl = String.format(
+                "http://%s:%s/%s/districts/DW",
+                OpenApiTestConstants.DEFAULT_HOST_NAME,
+                OpenApiTestConstants.DEFAULT_ENDPOINT_PORT,
+                DEPLOYMENT_NAME);
+        //  MP OpenAPI up
+        onlineManagementClient = ManagementClientProvider.onlineStandalone();
+        OpenApiServerConfiguration.enableOpenApi(onlineManagementClient);
+    }
+
+    @AfterClass
+    public static void tearDown() throws ManagementClientRelatedException, IOException {
+        //  MP OpenAPI down
+        try {
+            OpenApiServerConfiguration.disableOpenApi(onlineManagementClient);
+        } finally {
+            onlineManagementClient.close();
+        }
+    }
+
+    @Deployment(testable = false)
+    public static Archive<?> createCDeployment() {
+        WebArchive deployment = ShrinkWrap.create(
+                WebArchive.class,
+                String.format("%s.war", DEPLOYMENT_NAME))
+                .addClasses(RestApplication.class)
+                .addClasses(
+                        District.class,
+                        DistrictEntity.class,
+                        DistrictService.class,
+                        InMemoryDistrictService.class,
+                        DistrictsResource.class,
+                        RoutingServiceConstants.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return deployment;
+    }
+
+    /**
+     * @tpTestDetails Tests for application endpoint to be working
+     * @tpPassCrit Verifies that the returned JSON data corresponds to server side POJO
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @InSequence(1)
```

Comment:
could you provide more info to reason `@InSequence` in the test case?

Reply:
Yes. I believe that from testing perspective it makes sense to check the endpoint itself before verifying that the generated OpenAPI doc is actually reflecting its documentation.

Reply:
But other tests are executed no matter of the result of this test,right?

Reply:
Well, yes, I get your point indeed, this was actually just meant to give a "logical" ordering to tests, but I agree with what you say. I removed the annotation because your reasoning makes sense as the ordering would not actually affect the workflow, thanks.